Shadows.md

```diff
@@ -0,0 +1,27 @@
+
+# Comparison of the Shadow blur/radius property across platforms
+
+The shadow blur-radius property on Web, shadowRadius property on Apple platforms (iOS/macOS), and elevation property on Android all affect how blurred out the edges of a shadow look. However, they each affect the resulting shadow differently, and setting the same value for each of these will not result in a shadow that looks the same across platforms.
+
+## Convention
+
+If you are working with shadowRadius on an Apple platform trying to align with designs made with a web tool (like Figma or Sketch), divide the blur by 2.  This applies to React Native as well.
+This applies to these platforms on React Native as well (React Native, React Native macOS, and React Native Web)
+
+## Rationale
+
+Here is a comparison of shadow blur/radius across different platforms:
```

Comment:
Maybe something more like 

"We compared how shadows are rendered across Platforms, and found a discrepancy in how blur is handled. For whatever reason, divine by 2 seems to have the correct effect. See the table below:

And you can either keep the table in this section, or move it to examples. I'm not too strongly opinionated either way.